akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +58,11 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    CompletionStage<Done> ignoreDataBytesJ(Materializer materializer);
```

Comment:
We do for getters `thing: String` vs. `getThing: String`.
This is the first ever time we have a `run*` that's mixing java and scala actually - since we keep those APIs separate usually, except the HTTP Model structure as you can see.